docs/labwc-theme.5.scd

```diff
@@ -126,6 +126,32 @@ This syntax is not documented on the openbox.org wiki, but is supported by
 openbox and is used by many popular themes. For the sake of brevity, these
 elements are not listed here, but are supported.
 
+*window.active.shadow.enabled*
```

Comment:
This sounds strange, conditions in a theme. Couldn't those be replaced simply by a check for  presence or not of `window.*.shadow.*` ?

Reply:
Sure, can avoid having the boolean in the theme if that would be weird.

@Consolatis suggested putting the on/off in rc.xml and having the other settings in themerc.  To me it feels a bit odd to have them in separate files, but I'm not very familiar with labwc config so I will follow other people's lead here.

Reply:
I'm fine with the `.enabled` in themerc. I think it's more explicit and obvious to users compared with parsing `.color` and others for magic-numbers, `-1`, `INT_MIN`, etc.

I suspect @Consolatis's thought on `rc.xml` was to have a bigger override for modest hardward, but we could always add that later.

Reply:
> I suspect @Consolatis's thought on `rc.xml` was to have a bigger override for modest hardward

Right, with the enabled option being in the theme it is the theme used that decides about a potential performance impact. So the user would need to use `themerc-override` to "fix" that which might not be obvious. (Thinking longer term here, no theme currently sets those properties because, well, they do only exist within this PR for now.)

> but we could always add that later

Agree.